### PR TITLE
OEL-3422: Added aria-label on language switcher link.

### PR DESCRIPTION
**Pull request by tibi2303**

### CODE REVIEW

File: templates/overrides/navigation/links--language-block.html.twig [hunk 1]
```diff
@@ -9,31 +9,26 @@
...
-) %}
-  {% set title = _language.link.label %}
-  {% set link_title %}
-    <span class="d-block d-lg-inline-block">{{ title }}</span>
-  {% endset %}
+{% set extra_attributes = create_attribute({
+  'class': ['text-center'],
+  'data-bs-toggle': 'modal',
+  'data-bs-target': '#' ~ _language_link.target,
+  'aria-label': ('Change language. Current language is '|t) ~ _language_link.label ~ '.',
```
Comment (drishu) [inline templates/overrides/navigation/links--language-block.html.twig]: this is not the way to use the t function, please add a placehoder and set _language_link.label as parameter

File: tests/src/Kernel/MultilingualBlockTest.php [hunk 2]
```diff
@@ -87,6 +87,7 @@ public function testBlockRendering(): void {
     $link = $crawler->filter('div.language-switcher > a');
     $this->assertSame('English', trim($link->text()));
     $this->assertSame('#', $link->attr('href'));
+    $this->assertSame('Change language. Current language is English.', $link->attr('aria-label'));
```
Comment (drishu) [inline tests/src/Kernel/MultilingualBlockTest.php]: set another language and assert the language changes

**Review by drishu — CHANGES_REQUESTED** [review]

File: tests/src/Kernel/MultilingualBlockTest.php (hunk 2 above)
Comment (tibi2303) [reply tests/src/Kernel/MultilingualBlockTest.php]: added

File: templates/overrides/navigation/links--language-block.html.twig (hunk 1 above)
Comment (tibi2303) [reply templates/overrides/navigation/links--language-block.html.twig]: yes, thanks

**Review by drishu — APPROVED** [review]

**Pull request merged**